test/cli.py

```diff
@@ -533,6 +533,54 @@ def test_011_cli_benchmark(self):
 
         assert_success_of_builds([test_script], None, check_perf=True)
 
+    def test_012_cli_labels(self):
+
+        # Only build models labels with test_group::a
+        testargs = [
+            "benchit",
+            "benchmark",
+            bash(f"{corpus_dir}/*.py"),
+            "--label",
+            "test_group::a",
+            "--build-only",
+            "--cache-dir",
+            cache_dir,
+        ]
+        with patch.object(sys, "argv", flatten(testargs)):
+            benchitcli()
+
+        assert len(cache.get_all(cache_dir)) == 1
```

Comment:
I'm nervous that we're adding more tests that make assumptions about the contents of the test corpus, which is shared amongst all tests. For example, if I were to use linear.py as a starting point for a new input script for the corpus, then that new script would also have `test_group::a` at the top of it. 

Is there a way to code this test so that it doesn't have a baked in assumption about the number of scripts with `test_group::a`?

Reply:
Modified to ensure that we will not break this test if we accidentally add another test with an existing `test_group` label. 

Let me know what you think. 

We can also revert that change (to be less verbose) and simply do not add the `test_group` label to other test scripts in the future.